File: 6EGE/aa.cpp
#include <iostream>
using namespace std;

bool fu(int s){
    s /= 15;
    int n = 14;
    while( s < 285){
        if((s + n) % 9 == 0){
            s += 11;
        }
        n += 13;
    }
    return n == 118;

}

int main(){
    int kol = 0;
    for(int i = -1400; i <= 150000; i++){
        if(fu(i))
            kol ++;
    }

    cout << kol;

    return 0;
}

File: 14EGE/aa.py
a = (2 * 343 ** 123 + 2401) * (3 * 343 ** 137 - 2401)
kol = 0
while a > 0:
    if a % 7 == 6:
        kol += 1
    a //= 7
print(kol)